scss/header.scss

```diff
@@ -90,3 +92,117 @@ body.activate-account-page {
     background: var(--background-color);
   }
 }
+
+// attempt to center the title
+// by mirroring the body grid
+
+.has-sidebar-page {
+  .d-header > .wrap {
+    .contents {
+      display: grid;
+      grid-template-columns: var(--d-sidebar-width) minmax(0, 1fr);
```

Comment:
I genuinely don't get how this can work (yet it does) with only 2 cols defined. There's at least 3 visible (logo/title/icon+avatar).